ui/app/src/components/Header.tsx

```diff
@@ -11,17 +11,99 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { AppBar, Switch, Toolbar, Typography } from '@material-ui/core';
+import {
+  AppBar,
+  Box,
+  Button,
+  CircularProgress,
+  Divider,
+  Menu,
+  MenuItem,
+  Switch,
+  Theme,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import { ChevronDown } from 'mdi-material-ui';
+import { SxProps } from '@mui/system/styleFunctionSx/styleFunctionSx';
+import { MouseEvent, useState } from 'react';
+import { ProjectModel, useProjectQuery } from '../model/project-client';
+import Toast from './Toast';
+
+function ProjectMenu(props: { projectList: ProjectModel[] }): JSX.Element {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleMenu = (event: MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleCloseMenu = () => {
```

Comment:
I'd just inline these two single-line functions in the single place where they're used?

Reply:
Ok `handleCloseMenu()` is actually used in two places... but meh.

Reply:
I think it won't be used anymore once we have the code to switch from a workspace to another one (with routing and so one)